src/main/java/pw/react/backend/controller/FlatController.kt

```diff
@@ -138,7 +138,7 @@ class FlatController(
         )
         //val newAddress = addressService.save(address)
 
-        val newFlat = newFlatDto.toDomain(address,flatOwner)
+        val newFlat = newFlatDto.toDomain(address,flatOwner, timeProvider().toLocalDateTime(TimeZone.currentSystemDefault()).date.toJavaLocalDate())
```

Comment:
Tego javowego `LocalDate` bym używał w ostateczniści bo musisz przez to dużo pisać. Lepiej korzystaj z `LocalDate` z libki `kitlinx-datetime`. Tylko encje z jpa muszą mieć ten javowy localdate

Reply:
już po ptokach bywa